lms/static/sass/views/_program-marketing-page.scss

```diff
@@ -1,1767 +1,179 @@
+      img{
+        height:100px;
+        width:100px;
+        border-radius:100px;
       }
-      .course-index .accordion .course-navigation .button-chapter .group-heading {
-        padding: 15px 40px 15px 20px;
-      }
-      .ui-state-default.slick-header-column {
-        float: right !important;
-        left: inherit !important;
-        width: 84px !important;
-        height: 25px;
-        padding: 5px;
-      }
-      .grid-canvas {
-        width: 1097px !important;
-      }
-      .instructor-dashboard-content-2 {
-        .slickgrid .slick-cell {
-          float: right;
-          right: 0;
-          left: inherit;
-          position: relative !important;
-          width: 84px;
 
-          &.l0.r0 {
-            border-right: 1px dotted silver;
-          }
-          &.l12.r12 {
-            border-right: 0;
-          }
-        }
-        .content-history-table-inner .slickgrid .slick-cell {
-          width: 219px;
-        }
-        .report-downloads-table .slickgrid .slick-cell {
-          width: 100%;
-        }
-      }
-      .slick-header-columns {
-        right: 0 !important;
-        left: inherit !important;
-        width: 1097px !important;
-      }
-      .content-history-table-inner .ui-state-default.slick-header-column {
-        width: 219px !important;
+      .instructor-name{
+        font-weight: bold;
       }
-    }
 
-    @media(min-width: 768px) {
-      header.header-main .list-inline li.user-account ul:before {
-        right: 0.4rem;
-      }
-      .course-card{
-        display: inline-block;
-        margin: 0 calc(0.5 * 2.6rem) 30px calc(0.5 * 1rem) !important;
-        width: calc(50% - 22px);
-        vertical-align: top;
-        position: absolute;
-        height: 283px;
-        max-height: 283px;
-        overflow: hidden;
-        &:nth-child(even){
-          margin-right: 0 !important;
-        }
-        .title{
-          a{
-             height: auto;
-             max-height: 50px;
-             overflow: hidden;
-             display: block;
-             color: #005686;
-             text-decoration: underline;
-          }
-          small{
-            margin-top: 10px;
-            > span{
-              font-size: 0.9rem;
-            }
-          }
-        }
-        p{
-          margin-bottom: 30px;
-          min-height: 75px;
-          max-height: 75px;
-          overflow: hidden;
-          position: absolute;
-          bottom: 66px;
-          left: 20px;
-          right: 20px;
-        }
-        .btn-enroll{
-          display: inline-block;
-          background-color: $green !important;
-          min-width: 220px;
-          padding: 10px 30px;
-          font-size: 16px !important;
-          font-weight: 600 !important;
-          text-decoration: none;
-          position: absolute;
-          bottom: 20px;
-          left: 20px;
-        }
+      .instructor-position{
+        font-weight: bold;
       }
     }
+  }
+  #faqs{
```

Comment:
I also don't love targeting ids. What is special about this section? Can all questions just be styled the same?